packages/terra-progress-bar/src/ProgressBar.module.scss

```diff
@@ -18,7 +18,7 @@
     background: #d3d4d5;
     border-radius: var(--terra-progress-bar-border-radius, 0.5em);
     display: block;
-    height: var(--terra-progress-bar-height-small, 0.8rem);
+    height: var(--terra-progress-bar-small-height, 0.8rem);
```

Comment:
Do we want to keep the `small` identifier here? Similar situation as the Badge.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/febefb0fcb70b1260987d9b4a6758f730eadba72